ParabolInc/parabol
# Animate remote reflection when it enters or leaves Spotlight

fix #5489

**How to test:**

- Create a retro meeting
- Add various reflections
- Go to **Group** phase
- Join the meeting from another browser window as User B
- As User A, open a Spotlight on any reflection
- As User B, drag a reflection that doesn’t exist in the Spotlight and hover it above a reflection that does exist in the Spotlight. As it’s hovering, User A should now see the remote reflection hovering above the Spotlight with a fade-in animation
- As User B, hover a reflection above a reflection that does **not** exist in the Spotlight. User A should now see the remote reflection disappearing from the Spotlight with a fade-out animation.
- As User B, drop the reflection not in a group so its returns to the Kanban. As user A, see it disappears with a fade-out animation.
